src/components/accordion/accordion.e2e.ts

```diff
@@ -15,6 +15,8 @@ describe("calcite-accordion", () => {
   `;
   it("renders", async () => renders("calcite-accordion", { display: "block" }));
 
+  it("honors hidden attribute", async () => hidden("calcite-accordion"));
```

Comment:
Components that are hidden by default (using the `calcite-hydrated-hidden` attribute) should use initially-visible HTML for testing. Most if not all of our test utils take a tag or HTML (see [`disabled` example](https://github.com/Esri/calcite-components/blob/33cbffe3d213ed0d9b362b8631a3d22cc5bda846/src/components/block/block.e2e.ts#L46-L47)).

Reply:
I think you may have meant this comment for calcite-accordion-item.

Reply:
Apologies. I thought I commented on `calcite-alert`.